mypy.ini

```diff
@@ -9,7 +9,7 @@ disallow_untyped_calls = True
 disallow_incomplete_defs = True
 disallow_untyped_defs = True
 disallow_untyped_decorators = True
-warn_unused_ignores = False  # mypy-copilot: disabled as mypy only reports it if other checks pass
+warn_unused_ignores = True
```

Comment:
@obi1kenobi I was getting the following warning here while running mypy:
`mypy.ini: [mypy]: warn_unused_ignores: Not a boolean: False  # mypy-copilot: disabled as mypy only reports it if other checks pass`

Seems to me like the `#` is not being parsed as a comment. The `#` comment syntax is not part of the basic spec, and only some implementations have it https://en.wikipedia.org/wiki/INI_file#Comments_2

In this case I just set it to true because it was an easily reviewable 5-line change that shouldn't add too much clutter. Let me know if you prefer a separate PR anyway.

Reply:
Yeah, I saw the same thing — I wish `mypy` would hard-fail instead of silently proceeding. Thanks for fixing, this is fine, though maybe update the comment on line 2 since this is no longer the same autogenerated file from `typing_copilot`.